packages/core/src/hooks/multiChainCalls.test.tsx

```diff
@@ -0,0 +1,52 @@
+import { MockProvider } from '@ethereum-waffle/provider'
```

Comment:
Would also be good if you added similar small multi-chain tests for easy hooks like useEtherBalance